config/initializers/user_action_events.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
```

Comment:
I think we should leave out the auto creation of these and the yml files for now. It will result in an error in any environment that doesn't have these events, but that should be fine (the logger will catch the error). I want us to harden this implementation before it's live on other people's machines and on the stacks, otherwise it could be a headache as we add/change entries in the `user_action_events.yml` file

Reply:
(by leave out really I just mean let's pull this and put in another PR and review it separately)